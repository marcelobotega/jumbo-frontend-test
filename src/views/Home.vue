<template>
  <div class="home">
    <Stores />
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
import Stores from "@/components/Stores.vue";

@Component({
  components: {
    Stores,
  },
})
export default class Home extends Vue {}
</script>
