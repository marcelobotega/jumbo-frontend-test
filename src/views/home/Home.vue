<template>
  <div class="home-wrapper content-section">
    <h1>Welcome</h1>
    <p class="mt-1">Use the menu above to navigate through the system</p>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";

@Component
export default class Home extends Vue {}
</script>
