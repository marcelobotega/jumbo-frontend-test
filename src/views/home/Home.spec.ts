import { shallowMount } from "@vue/test-utils";
import Home from "@/views/home/Home.vue";

describe("Home.vue", () => {
  it("should create home", () => {
    const wrapper = shallowMount(Home);
    expect(wrapper).toMatchSnapshot();
  });
});
