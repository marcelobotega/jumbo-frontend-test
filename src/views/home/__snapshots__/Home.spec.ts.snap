// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue should create home 1`] = `
<div class="home-wrapper content-section">
  <h1>Welcome</h1>
  <p class="mt-1">Use the menu above to navigate through the system</p>
</div>
`;
